steps/cordova-android/0.9.0/step.yml

```diff
@@ -0,0 +1,48 @@
+title: Cordova Android
+summary: This step allows you to build cordova-based Android projects.
+description: |-
+  The Cordova CLI is the main tool to use for the cross-platform workflow
+  described in the [Overview](https://cordova.apache.org/docs/en/latest/guide/overview/index.html).
+website: https://github.com/donvigo/steps-cordova-install
+source_code_url: https://github.com/donvigo/steps-cordova-install
+support_url: https://github.com/donvigo/steps-cordova-install/issues
+published_at: 2016-02-13T18:42:15.47182447+02:00
+source:
+  git: https://github.com/donvigo/steps-cordova-android.git
+  commit: 18e9dd608cdb9d6051e502480b3f01583bc0d17b
+host_os_tags:
+- ubuntu
+project_type_tags:
+- android
+type_tags:
+- cordova
+- build
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- cordova_command: build
+  opts:
+    is_expand: true
```

Comment:
is_expand: true is the default value of is_expand, so you can remove every is_expand: true.